src/App.module.scss

```diff
@@ -1,38 +1,41 @@
-.App {
-  text-align: center;
-}
+@import url('https://fonts.googleapis.com/css2?family=Orbitron&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Orbitron&family=Rajdhani&display=swap');
 
-.appLogo {
-  height: 40vmin;
-  pointer-events: none;
+@mixin title($textcolor: white) {
```

Comment:
could we start using constants for the colors?
ie - something like

```scss
$colorTextLight: white;
$colorTextDark: black;

// elsewhere
@mixin title($textColor: $colorTextLight) {
```

That way, if we ever want to tweak the color scheme, we can just change those two constants in that one place and everything else automatically updates with it.